src/ast/dibuilderbpf.cpp

```diff
@@ -292,9 +292,16 @@ DIGlobalVariableExpression *DIBuilderBPF::createMapEntry(
 
 DIGlobalVariableExpression *DIBuilderBPF::createGlobalInt64(
```

Comment:
Nit: we probably don't need this wrapper function now. It's easy enough to write `createGlobalVariable(name, CreateInt64());`

Reply:
Fixed in latest PR update (coming soon)